web/libs/ui/src/shad/utils/index.ts

```diff
@@ -0,0 +1,6 @@
+import { type ClassValue, clsx } from "clsx";
+import { twMerge } from "tailwind-merge";
+
+export function cn(...inputs: ClassValue[]) {
```

Comment:
we already have `cn()` in BEM, `clsx()` in other places and this new `cn()` without description. what is this?

Reply:
it's part of shadcn. cn is basically clsx + tailwind-merge to correctly merge list of class names